# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux24benh/96-quota-for-userspace.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Samba needs the struct ...

 - Rene Rebe <rene@exactcode.de>

--- linux-2.4.24/include/linux/quota.h.orig	2004-01-09 21:51:11.000000000 +0100
+++ linux-2.4.24/include/linux/quota.h	2004-01-09 21:55:10.000000000 +0100
@@ -132,12 +132,6 @@
 	__u32 dqi_valid;
 };
 
-#ifdef __KERNEL__
-
-#include <linux/dqblk_xfs.h>
-#include <linux/dqblk_v1.h>
-#include <linux/dqblk_v2.h>
-
 /*
  * Data for one user/group kept in memory
  */
@@ -152,6 +146,12 @@
 	time_t dqb_itime;	/* time limit for excessive inode use */
 };
 
+#ifdef __KERNEL__
+
+#include <linux/dqblk_xfs.h>
+#include <linux/dqblk_v1.h>
+#include <linux/dqblk_v2.h>
+
 /*
  * Data for one quotafile kept in memory
  */
